cmd/autonity/misccmd.go

```diff
@@ -199,45 +237,40 @@ func genOwnershipProof(ctx *cli.Context) error {
 	}
 
 	treasury := args[0]
-	data, err := hexutil.Decode(treasury)
-	if err != nil {
-		utils.Fatalf("Failed to decode: %v", err)
-	}
-	// Add ethereum signed message prefix to maintain compatibility with web3.eth.sign
-	// refer here : https://web3js.readthedocs.io/en/v1.2.0/web3-eth-accounts.html#sign
-	prefix := fmt.Sprintf("\x19Ethereum Signed Message:\n%d", len(data))
-	hash := crypto.Keccak256Hash([]byte(prefix), data)
-	//sign the data hash
-	nodeSignature, err := crypto.Sign(hash.Bytes(), nodePrivateKey)
+	signatures, err := crypto.AutonityPOPProof(nodePrivateKey, oraclePrivateKey, treasury, consensusKey)
 	if err != nil {
-		utils.Fatalf("Failed to sign: %v", err)
-	}
-	oracleSignature, err := crypto.Sign(hash.Bytes(), oraclePrivateKey)
-	if err != nil {
-		utils.Fatalf("Failed to sign: %v", err)
+		if err == hexutil.ErrMissingPrefix {
+			utils.Fatalf("Failed to decode: hex string without 0x prefix")
+		}
+		utils.Fatalf("Failed to generate Autonity POP: %v", err)
 	}
-	multisig := append(nodeSignature[:], oracleSignature[:]...)
-	hexStr := hexutil.Encode(multisig)
-	fmt.Println("Signature hex:", hexStr)
+
+	fmt.Println("Node consensus key:", consensusKey.PublicKey().Hex())
+	fmt.Println("Signatures:", hexutil.Encode(signatures))
```

Comment:
```suggestion
	fmt.Println(hexutil.Encode(signatures))
```
Make it easier to script.